src/main/java/uk/gov/hmcts/reform/cwrdapi/client/domain/CaseWorkerProfile.java

```diff
@@ -45,9 +47,10 @@
 
     @MappingField(clazz = Location.class, objectCount = 2)
     @NotEmpty(message = "no primary or secondary location exists")
-    @JsonProperty("base_location")
+    @JsonProperty("baseLocation")
```

Comment:
Any reason for this change? As https://tools.hmcts.net/confluence/pages/viewpage.action?pageId=1457304753&__ncforminfo=IzLDf-_makrWFICHZoVkLGat0QmmYqoQCCb-9qTWNZbIdYXsKre2jr18z8mektl-4t1wCj7scLzTqHzQAjvFWx3i1kXD9TBb says the response should have `base_location `

Reply:
Updated the code to use "base_location", as specified in the document.

Reply:
thanks